tests/Microsoft.Identity.Test.Integration.net45/Infrastructure/SeleniumExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static IWebDriver CreateDefaultWebDriver()
 
             // ~2x faster, no visual rendering
             // remove when debugging to see the UI automation
-            options.AddArguments("headless");
+            //options.AddArguments("headless");

```

Comment:
I suppose the comment should say, remove // from the line below to see the UI? Also I suspect you didn't intend to change this particular line?

Reply:
No, didn't mean to change this. Fixed.